src/main/kotlin/com/github/shiraji/ipgwizard/step/IPGWizardSupportLanguageStep.java

```diff
@@ -18,19 +18,24 @@
     private WizardContext context;
     private IPGWizardBuilder builder;
 
-    private List<String> languages = new ArrayList<String>() {{
-        add("Java");
-        add("Kotlin");
-    }};
+    public enum Language {
+        JAVA("Java"),
+        KOTLIN("Kotlin");
+
+        private final String name;
+
+        Language(String name) {
+            this.name = name;
+        }
+    }
 
     static {
     }
 
     public IPGWizardSupportLanguageStep(WizardContext context, IPGWizardBuilder builder) {
         this.context = context;
         this.builder = builder;
-
-        for (String language : languages) {
+        for (Language language : Language.values()) {
```

Comment:
Since now it has Enum, let's use it advantages. Replacing for-loop with following line.

```java
languageComboBox.setModel(new DefaultComboBoxModel(Language.values()));
```

To make it consist, the combobox selection should be "Java" and "Kotlin". So Enum needs to override `toString()`

```java
    public enum Language {
        JAVA("Java"),
        KOTLIN("Kotlin");

        private final String name;

        Language(String name) {
            this.name = name;
        }

        @Override
        public String toString() {
            return name;
        }
    }
```